apps/meteor/client/views/admin/apps/AppsPage.tsx

```diff
@@ -58,16 +58,29 @@ const AppsPage = ({ isMarketplace }: AppsPageProps): ReactElement => {
 					)}
 				</ButtonGroup>
 			</Page.Header>
-			<Tabs bg={colors.n100}>
-				<Tabs.Item onClick={(): void => marketplaceRoute.push({ context: '' })} selected={isMarketplace}>
+			<Tabs bg={colors.n100} borderBlockEnd='4px solid #CBCED1' marginInline='x24'>
```

Comment:
dont just, dont  pass styles to components, if you want, create this divider inside the fuselage lib and than use as some kind of variation

Reply:
Will open a task to solve this since waiting for fuselage would unnecessarily stall this PR.